README.md

```diff
@@ -4,7 +4,20 @@
 [![Chat](https://img.shields.io/static/v1.svg?label=chat&message=mattermost&color=145dbf)](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind)
 
 # Codewind for VS Code
-Create and develop cloud-native, containerized web applications from VS Code.
+Codewind for VS Code provides tools to help build cloud-native, containerized web applications from VS Code, regardless of which IDE or language you use. Rapidly create an application from a template project and launch, update, test, and debug in Docker containers on the desktop.
+
+## Features
+- Create new containerized projects or add existing ones.
+- View your containerized projects, including the application and build status.
+- Debug Microprofile/Java EE and Spring applications.
+- Set up a debug session in a Chromium-based browser for Node.js applications.
+- View application and build logs in the Console view.
+- Integrate validation errors into the Markers view.
```

Comment:
"See validation errors in the Markers view" seems more accurate. "Integrate" makes it sound like developers themselves would have some ability to integrate validation errors.